tap_clickup/schemas/task.json

```diff
@@ -226,7 +226,7 @@
                             "type": ["string", "boolean", "null"]
                         },
                         "value": {
-                            "type": ["integer", "string", "boolean", "null", "array", "object", "number"]
+                            "type": ["integer", "string", "null", "array", "object", "number"]
```

Comment:
The value can be a Boolean though which is why we have it listed in the types here. 

Strings being interpreted incorrectly as boolean is an issue though!

Reply:
The way I found out about this isse was as follows, maybe you can see it replicated by doing this, maybe this helps to at least see this happening: 
1 - I created a task in ClickUp with all types of custom fields and set them arbitrarily. This way I had a sample of all possible values and types;
2 - changed the client.py and added this snippet:
```
...
    def parse_response(self, response: requests.Response) -> Iterable[dict]:
       """Parse the response and return an iterator of result rows."""
       yield from extract_jsonpath(self.records_jsonpath, input=response.json())
       
    import json
    def parse_response(self, response: requests.Response) -> Iterable[dict]:
        # Save the raw response data before parsing
        with open("raw_data_before_parse.json", "w") as raw_data_file_before:
            raw_data_file_before.write(response.text)

        raw_records = list(extract_jsonpath(self.records_jsonpath, input=response.json()))  # Convert generator to a list
        
        # Save the raw records after parsing
        with open("raw_records_after_parse.json", "w") as raw_data_file_after:
            raw_data_file_after.write(json.dumps(raw_records))

        for raw_record in raw_records:
            record = raw_record.copy()  # Create a copy of the raw record

            # Manually convert the "value" field to a string if it exists
            if "value" in record:
                try:
                    record["value"] = str(record["value"])
                except (ValueError, TypeError):
                    pass  # Handle conversion errors if necessary

            yield record
```
This way we have files showing one response from the API

Then we crosscheck with what was saved into w/e target we set.

We will see the aforementioned behaviour. Then by trial and error, by just removing the boolean from the task.json it started to get those values correctly.